### PR TITLE
Rework line display

### PR DESCRIPTION
**Pull request by fsimonjetz**
Adds logic and display schemas to pass additional information from the manuscript to the corpus ([trello card](https://trello.com/c/70pZysFU))

### CODE REVIEW

File: ebl/corpus/application/corpus.py [hunk 1]
```diff
@@ -107,9 +107,15 @@ def find_chapter_for_display(self, id_: ChapterId) -> ChapterDisplay:
         return self._inject_parallels(self._repository.find_chapter_for_display(id_))
 
     def find_line(
-        self, id_: ChapterId, number: int
+        self, id_: ChapterId, number: int, with_manuscript_joins: bool = False
     ) -> Tuple[Line, Sequence[Manuscript]]:
```
Comment (yCobanoglu) [inline ebl/corpus/application/corpus.py:111]: optional no boolean flag if possible

File: ebl/corpus/domain/line.py [hunk 2]
```diff
@@ -82,6 +82,7 @@ class LineVariant:
         converter=pydash.flow(set_enclosure_type, set_language)
     )
     note: Optional[NoteLine] = None
+    # manuscript_lines
     manuscripts: Sequence[ManuscriptLine] = tuple()
```
Comment (yCobanoglu) [inline ebl/corpus/domain/line.py:86]: no comments unless really necessary

Comment (yCobanoglu) [reply ebl/corpus/domain/line.py:86]: refactor to manuscripts_lines

Comment (fsimonjetz) [reply ebl/corpus/domain/line.py]: Turns out this would affect a considerable number of files - let's postpone this to a later stage.

**Pull request merged**